src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -22,6 +22,7 @@
 import math as _math
 import six as _six
 
+from ._tf_model_architecture import StyleTransferTensorFlowModel
```

Comment:
Won't this unconditionally import TensorFlow?

Reply:
fixed to import after use_tf is validated.